### ISSUE TITLE
Result Image(Sony Model -The green is scattered)

### ISSUE THREAD

**Issue by ghost0710** [cchen156/Learning-to-See-in-the-Dark#42]
I use your sony pretrainned model, Execute test_sony.py 
I wonder why green dots are spreading in the output image.
![image](https://user-images.githubusercontent.com/37354190/43062950-5caa83b2-8e95-11e8-8d06-ebe067c55677.png)

**Comment by cchen156** [comment]
Our results are not artifact free which was stated in the paper. If you train a model for the outdoor images only, these artifacts will disappear. Probably the indoor images are so different (e.g., light sources). When one network tried to process both indoor and outdoor images, the performance decreased.

**Comment by cchen156** [comment]
Yes. I trained a model for outdoor images only. The results are better.

**Comment by ghost0710** [comment]
please share your only outdoor model

**Comment by cchen156** [comment]
It is easy to train using the provided code. The first part of the data is outdoor images and the second part is indoor images.  I did not keep the outdoor model. You can use images 00000 to 00183 to train it.

**Comment by ghost0710** [comment]
![image](https://user-images.githubusercontent.com/37354190/43189281-2d0adc4a-9031-11e8-8bc9-010f5e7aa0b5.png)

There is a phenomenon that the image in Sony result is crushed like the above
What is the reason? Especially in very dark environments

**Comment by oneTaken** [comment]
@ghost0710 I tried many combinations, with different models and different super-parameters.And I found this image is always predicted like this. In my opinion, if this input area has very little information(the environment is very dark), the model would tried to suggest what's information here. So there are some blur areas.
